### ISSUE TITLE
Some weather status icons (ex: overcast clouds, rain drops) do not contrast well with some background colors

### ISSUE THREAD

**Issue by nekohayo** [amit9838/mousam#131]
If you turn off the colored backgrounds (for better contrast/readability etc.) the white part of the clouds is barely visible for someone with good eyesight, so they will be invisible to someone with bad eyesight:

![image](https://github.com/amit9838/mousam/assets/479401/50fd5df2-3c92-407a-aa8a-d3cb81681c0e) 
![image](https://github.com/amit9838/mousam/assets/479401/01268249-12ba-42dc-b049-6f8b0c1f9613)

Short of using symbolics, maybe those icons could benefit from some sort of dropshadow or border?

The icons might need to have specific light and dark variants too, because some elements are conversely hard to see in dark mode. For example, the raindrops here are difficult to notice as they are tiny and dark-blue… if I sit 60cm from my monitor, I literally cannot see them:

![image](https://github.com/amit9838/mousam/assets/479401/6ea362b6-b4e4-4a0b-bd90-0b9a88767545)

**Comment by amit9838** [comment]
Yes, that's a good solution for light theme. I'll look into it. I am also looking for good weather icons, which can fit with dynamic weather background, but not able to find. 

Would be nice if someone from the community could share some good weather icons.

**Comment by nekohayo** [comment]
Arguably, the whole problem goes away if you use symbolic weather icons, like GNOME's, as they would then have maximum contrast at all times against any background, and could be recolored just like the text, but I don't know if that's a road you'd like to take.

**Comment by amit9838** [comment]
I think  gnome coloured icons are not that good, on the top of that we have a dynamic bg.

I would prefer to use the monochrome icons when using dynamic bg, but with dynamic bg we need some good quality icons, and another option is to improve to the css gradient.
